packages/flutter_tools/test/general.shard/flutter_project_metadata_test.dart

```diff
@@ -50,6 +50,21 @@ void main() {
     expect(logger.traceText, contains('.metadata file at .metadata was empty or malformed.'));
   });
 
+  testWithoutContext('projectType is populated when version is null', () {
```

Comment:
This test crashes with `_CastError: Null check operator used on a null value` on master.